lib/rocket_chat/messages/channel.rb

```diff
@@ -40,6 +40,21 @@ def list(offset: nil, count: nil, sort: nil, fields: nil, query: nil)
         response['channels'].map { |hash| RocketChat::Room.new hash } if response['success']
       end
 
+      #
+      # channels.online REST API
+      # @param [String] room_id Rocket.Chat room id
+      # @return [Users]
+      # @raise [HTTPError, StatusError]
+      #
+      def online(room_id)
```

Comment:
Could you update this to use the same method signature as the `join` method?

ie `def online(room_id: nil, name: nil)` then pass the param to the body using `body: room_params(room_id, name)` helper. 

You'll need to update your params above and the README docs

Reply:
I'm not sure why I keep getting `NameError:
       undefined local variable or method `online' for #<RSpec::ExampleGroups::RocketChatMessagesChannel::Online::ValidSession:0x00007f8fff0ea0d8>` when stubbing the call in the specs... (I've likely missed something) so let me know